example-apps/kitchen-sink-example/source/app.routes.ts

```diff
@@ -1,11 +1,11 @@
-import { provideRouter, RouterConfig, RouterModule }  from '@angular/router';
+import { RouterModule }  from '@angular/router';
 
 import { KitchenSinkRoutes, KitchenSinkDeclarations }
  from './containers/kitchen-sink.routes';
 import { RouterRoutes, RouterDeclarations }
  from './components/router/router.routes';
 
-export const routes: RouterConfig = [
+export const routes = [
```

Comment:
Yeah, there seems to be no longer a type for this; or at least it isn't the same. I don't see why they would just change it, so I suspect it was just removed. I could look into that.